src/main.rs

```diff
@@ -50,7 +50,15 @@ fn main() {
         Green,
         Blue,
     }
-
+    // Use the enum variants
+    let primary = Color::Red;
+    let secondary = Color::Green;
```

Comment:
Let's also add the `Color` type here too.
```suggestion
    let primary: Color = Color::Red;
    let secondary: Color = Color::Green;
```